app/components/ui/persistent-collapsable-container/component.js

```diff
@@ -0,0 +1,19 @@
+import Ember from "ember";
+
+export default Ember.Component.extend({
+  preferencesService: Ember.inject.service(),
+
+  isClosedDynamicComputed: function() {
```

Comment:
I believe using `Ember.computed("preferencesService", "settingsKey", function(){})`

Will work the same and is the preferred method. Did that not work?